server/app-engine/gtfsrt.py

```diff
@@ -174,12 +177,32 @@ def make_trip_update(list):
 
     return entity
 
-def alert_is_current(alert):
+def alert_is_current(alert, include_future_alerts):
     print('alert_is_current()')
     if not('time_start' in alert and 'time_stop' in alert):
         return False
     now = int(time.time())
-    return now >= int(alert['time_start']) and now < int(alert['time_stop'])
+    # No start or stop time listed. Is current until deleted.
+    if(alert['time_stop'] == 0 and (alert['time_stop'] == 0)):
```

Comment:
You're checking `time_stop` twice. Also, it looks like you're using 0 to denote both the beginning of time and the end of time. May I suggest instead to use 0 for beginning of time and 2^31/1000 as the end of time? With symbolic constants of course

Reply:
The HTML datetime selector sets the value to zero when no time is selected - I opted to leave as is rather than setting to null (which would be more descriptive since 0 is technically 1970). Another option would be to set them to 0 and 2^31 in javascript, as you said, and then have simpler logic here in python. I kind of like that.

Re: time_stop twice, drat! good catch.